source/about/faq.rst

```diff
@@ -60,8 +67,10 @@ history page: :doc:`history`
 Can I run Bukkit Plugins with this?
 -----------------------------------
 
-Sadly no, not natively. Sponge is using its own API (Sponge API), while Bukkit is using the Bukkit API. However there is the
-possibility that a third-party plugin for Sponge implements the Bukkit API on top of Sponge API.
+Sadly no, not natively. Sponge is using its own API (Sponge API), while Bukkit is using the Bukkit API. While third parties
```

Comment:
`Sadly no, not natively.` remove? I don't think anyone here is disappointed about that, nor do we think a plugin that adds bukkit comp is a good idea